src/test/kotlin/three/consulting/epoc/repository/CustomerServiceIntegrationTest.kt

```diff
@@ -27,4 +27,26 @@ class CustomerServiceIntegrationTest : IntegrationTest() {
         val customer: CustomerDTO? = customerService.findCustomerForId(1000L)
         assertThat(customer).isNull()
     }
+
+    @Test
+    fun `added customer is found from database`() {
+        assertThat(false)
+        val customer = CustomerDTO(null,"Testit Oy", "Innovating new innovation!", enabled = true)
+        val addedCustomer: CustomerDTO? = customerService.createCustomer(customer)
+        if (addedCustomer != null) {
+            if (addedCustomer.id != null) {
+                val dbCustomer: CustomerDTO? = customerService.findCustomerForId(addedCustomer.id!!)
+                assertThat(addedCustomer).isEqualTo(dbCustomer)
+                assertThat(addedCustomer.created).isNotNull
```

Comment:
Well the service.createCustomer you're testing here already returns the created customer, so it's enough to assert that IMO. Also you could create a json file similarly as in the controller tests and use the jackson objectMapper to convert that string created from a file into an object. If you go with the objectmapper that needs to be defined as an `@Autowired lateinit var` in the base class, just like the service.